tests/03-auth.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { chai } from 'mochainon';
+import { ResinAuth } from '../lib/auth';
+import { TokenType } from '../lib/token';
+import apiKeyFixtures from './fixtures/api-keys';
+import jwtFixtures from './fixtures/jwts';
+
+const { expect } = chai;
+const IS_BROWSER = typeof window !== 'undefined';
+
+let dataDirectory;
+if (!IS_BROWSER) {
+	// tslint:disable-next-line no-var-requires
+	const settings = require('resin-settings-client');
+	dataDirectory = settings.get('dataDirectory');
+}
+
+const auth = new ResinAuth({ dataDirectory });
+
+describe('ResinAuth', () => {
+	before(() =>
```

Comment:
No, not really.

Reply:
If you make this a beforeEach though, you guarantee every test starts with a clean slate, and you make it way easier to ensure these don't start depending on one another later on. And since this should be an unnoticeably quick operation, there's no real downside.

As a bonus, if you make this one a beforeEach, you can remove the `after` that I commented on below completely, because you'll have guaranteed that every test gets an empty token before it starts anyway.

Reply:
Ok done.